tests/integration/autoloader/autoload-with-broken-composer-01/vendor/composer/installed.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+This file is needed by real Composer's runtime API, and agent verifies its presence to determine if Composer is used.
+This file contains phpized version of composer.json for the project and its dependencies.
+*/
+  return array(
+    // Mocked data: root package
+    'root' => array(
+        'pretty_version' => 'v1.0.0',
+        'version' => '1.0.0.0',
+        'type' => 'project'
+    ),
+    // Mocked data: installed packages and their versions
+    'versions' =>  array(
```

Comment:
What about adding some tests where the package name is null version is valid, version is null name is valid, both version/package are null.

Reply:
Added in 5a73606d8399bcf6199bbdef4efb58bed536af6d.